miden-lib/asm/miden/kernels/tx/tx.masm

```diff
@@ -58,11 +58,15 @@ const.ERR_NON_FUNGIBLE_ASSET_ALREADY_EXISTS=0x00020051
 # EVENTS
 # =================================================================================================
 
-# Event emitted when a new note is created.
-const.NEW_NOTE_EVENT=131077
+# Event emitted before a new note is created.
+const.BEFORE_NEW_NOTE_EVENT=131083
+# Event emitted after a new note is created.
+const.AFTER_NEW_NOTE_EVENT=131084
 
-# Event emitted when an ASSET is added to a note
-const.BEFORE_ADD_ASSET_TO_NOTE_EVENT=131079
+# Event emitted before an ASSET is added to a note
+const.BEFORE_ADD_ASSET_TO_NOTE_EVENT=131085
+# Event emitted after an ASSET is added to a note
+const.AFTER_ADD_ASSET_TO_NOTE_EVENT=131086
```

Comment:
Shouldn't these names be consistent with the ones used in Rust? e.g., `NOTE_BEFORE_ADD_ASSET_EVENT` instead of `BEFORE_ADD_ASSET_TO_NOTE_EVENT`?